cloudformation/stacks/AutoSpotting/template.json

```diff
@@ -26,6 +31,21 @@
       "Default": "7",
       "Description": "Number of days to keep the Lambda function logs in CloudWatch.",
       "Type": "Number"
+    },
+    "MinOnDemandNumber": {
+      "Default": "0",
+      "Description": "Minimum on-demand instances (absolute number) to be kept running in your ASGs. It is a global default value that can be overridden on a per-group level using the autospotting_on_demand_number tag. It takes precedence over MinOnDemandPercentage, so it doesn't make sense to pass both of them.",
+      "Type": "Number"
+    },
+    "MinOnDemandPercentage": {
+      "Default": "0.0",
+      "Description": "Minimum on-demand instances (in percentage of the group capacity) that will be kept running in your ASGs. It is also a global default value that can be overridden on a per-group level using the autospotting_on_demand_percentage tag. MinOnDemandNumber takes precedence if both these parameters are passed",
+      "Type": "Number"
+    },
+    "Regions": {
+      "Default": "",
+      "Description": "Space separated list of regions where it should run, in case you may want to limit it to a smaller set of regions. If unset it will run against all available regions.",
+      "Type": "String"
```

Comment:
I tried that, but CloudFormation fails to set any value containing commas: http://docs.aws.amazon.com/lambda/latest/dg/API_Environment.html#SSS-Type-Environment-Variables

Commas are nicer when defined for manual local execution, since they don't need quotes.